app/src/main/java/com/keylesspalace/tusky/receiver/SendStatusBroadcastReceiver.kt

```diff
@@ -117,7 +117,8 @@ class SendStatusBroadcastReceiver : BroadcastReceiver() {
                 builder.setCategory(NotificationCompat.CATEGORY_SOCIAL)
                 builder.setOnlyAlertOnce(true)
 
-                notificationManager.notify(notificationId, builder.build())
+                // There is a separate "I am sending" notification, so simply remove the handled one.
+                notificationManager.cancel(notificationId)
```

Comment:
I wonder if there are more locations (special notifications) like this where notifications would not be removed now.

Upon creation (`NotificationHelper#make`) there are however only two special cases (compose and quick reply).